hooks/pre_exit/20-certs_regenerate.rb

```diff
@@ -1,6 +1,9 @@
 if module_enabled?('certs') && param('certs', 'regenerate').value == true
-  answers = kafo.config.answers
-  answers['certs']['regenerate'] = false
+  kafo.config.modules.each do |mod|
+    if mod.identifier == 'certs'
+      mod.params_hash['regenerate'] = false
+    end
+  end
 
-  kafo.config.store(answers)
+  kafo.send(:store_params)
```

Comment:
I agree this is a bit ugly, calling into a private method. My goal here is to patch the problem first with a fix that can be cherry picked cleanly backward.